data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/KafkaSourceConfig.java

```diff
@@ -51,6 +51,26 @@ public class KafkaSourceConfig {
     @JsonProperty("acknowledgments_timeout")
     private Duration acknowledgementsTimeout = DEFAULT_ACKNOWLEDGEMENTS_TIMEOUT;
 
+    @JsonProperty("serde_format")
+    private String serdeFormat;
+
+    @JsonProperty("client_dns_lookup")
+    private String clientDnsLookup;
+
+    @JsonProperty("ssl_endpoint_identification_algorithm")
+    private String sslEndpointIdentificationAlgorithm;
```

Comment:
Why is this needed? Where is this used. I have added https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/source/InsecureSslEngineFactory.java that can be used for insecure client.

Reply:
This configuration "ssl_endpoint_identification_algorithm" is optional and removed from the config file